lib/client/bst-config.ts

```diff
@@ -155,12 +195,17 @@ export class BSTConfig {
     private static async createConfig(nodeID?: string, sourceID?: string): Promise<any> {
         const lambdaConfig = LambdaConfig.defaultConfig().lambdaDeploy;
         const pipeInfo = await BSTConfig.createExternalResources(nodeID, sourceID);
-
+        const bstMessages = new BstMessages();
```

Comment:
this should call this.fetchMessages too to avoid crashing if the messages endpoint fail